third_party/move/move-prover/bytecode-pipeline/src/number_operation_analysis.rs

```diff
@@ -59,7 +59,7 @@ impl NumberOperationProcessor {
     /// Entry point of the analysis
     fn analyze<'a>(&self, env: &'a GlobalEnv, targets: &'a FunctionTargetsHolder) {
         self.create_initial_exp_oper_state(env);
-        let fun_env_vec = FunctionTargetPipeline::sort_targets_in_topological_order(env, targets);
+        let fun_env_vec = FunctionTargetPipeline::sort_in_reverse_topological_order(env, targets);
```

Comment:
Should we use a reverse iteration over the fun_env_vec, since it's now in reverse topological order?

Reply:
So, the iteration order has been a reverse topological order over the call graph, even in the previous code (so that you could visit the call graph in a bottom-up order). I just renamed the function to make this aspect clear.